bot/exts/moderation/defcon.py

```diff
@@ -153,6 +156,10 @@ async def _defcon_action(self, ctx: Context, days: int, action: Action) -> None:
                     }
                 }
             )
+
+            self.days = timedelta(days=days)
+            self.update_notifier()
```

Comment:
This looks like it'd be more appropriate for an `else` clause rather than being in the try block, but it can probably stay.